addons/stock/tests/common.py

```diff
@@ -78,14 +84,14 @@ def setUpClass(cls):
         cls.MoveObj = cls.env['stock.move']
         cls.LotObj = cls.env['stock.lot']
         cls.StockLocationObj = cls.env['stock.location']
-
         # Warehouses
         cls.warehouse_1 = cls.env['stock.warehouse'].create({
             'name': 'Base Warehouse',
             'reception_steps': 'one_step',
             'delivery_steps': 'ship_only',
             'code': 'BWH',
             'sequence': 5,
+            'company_id': cls.company.id,
```

Comment:
Didn't take a proper look at the whole PR, but wasn't the point of the change in `cls.env` to avoid adding to add these kind of lines? :thinking: